roles/kubernetes/preinstall/tasks/0070-system-packages.yml

```diff
@@ -95,6 +95,14 @@
   retries: 4
   delay: "{{ retry_stagger | random + 3 }}"
   with_items: "{{required_pkgs | default([]) | union(common_required_pkgs|default([]))}}"
-  when: not (ansible_os_family in ["CoreOS", "Container Linux by CoreOS"] or is_atomic)
+  when: not (ansible_os_family in ["CoreOS", "Container Linux by CoreOS", "ClearLinux"] or is_atomic)
   tags:
     - bootstrap-os
+
+- name: Install ipvsadm for ClearLinux
+  swupd:
+    name: ipvsadm
+    state: present
+  when:
+    - ansible_os_family in ["ClearLinux"]
```

Comment:
Should be: `["ClearLinux"] in ansible_os_family`

Reply:
this change is causing the below error. 
```
ERROR! Syntax Error while loading YAML.
  did not find expected '-' indicator

The error appears to have been in '/home/vagrant/kubespray/roles/kubernetes/preinstall/tasks/0070-system-packages.yml': line 107, column 22, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  when:
    - ["ClearLinux"] in ansible_os_family
                     ^ here
```
And it seems everywhere else the syntax is same as the one we have now. Pardon my ignorance, but am i missing something obvious here?

Reply:
Ah... Try this:

`"ClearLinux" in ansible_os_family`

Or keep it. Either will work, I believe.